game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/fire/firststrike/ClearFirstStrikeCasualties.java

```diff
@@ -89,9 +92,11 @@ public void execute(final ExecutionStack stack, final IDelegateBridge bridge) {
 
     final EnumSet<BattleState.Side> sidesToClear = getSidesToClear();
     final Collection<Unit> unitsToRemove =
-        new ArrayList<>(battleState.getWaitingToDie(sidesToClear));
+        new ArrayList<>(
+            battleState.getWaitingToDie(
+                sidesToClear.toArray(new BattleState.Side[sidesToClear.size()])));
```

Comment:
Codacy found an issue: [This call to Collection.toArray() may be optimizable](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6171481)